### ISSUE TITLE
Cannot boot all webservers of all examples at once

### ISSUE THREAD

**Issue by clarmso** [cypress-io/cypress-example-recipes#223]
**Steps:**
1. Clone the cypress-example-recipes repo via "git clone".
2. Run "cd cypress-example-recipes"
2. Run "npm start".

**Result:**
I constantly get the following output.  It looks like multiple examples try to run a test page on port 3000.  I cannot proceed until I type "Y" to run the server on a different port. One of the example is from `blogs__vue-vuex-rest`

? Something is already running on port 3000. Probably:
```
  node ../../node_modules/.bin/json-server --static . data.json --middlewares ../../node
_modules/json-server-reset (pid 63838)
  in /Users/clare/Documents/workspace/cypress-example-recipes/examples/blogs__vue-vuex-r
est

Would you like to run the app on another port instead? (Y/n) ? Something is already running on port 3000. Probably:
  node ../../node_modules/.bin/json-server --static . data.json --middlewares ../../node
_modules/json-server-reset (pid 63838)
  in /Users/clare/Documents/workspace/cypress-example-recipes/examples/blogs__vue-vuex-r
est
```

**Notes**
The following examples' cypress.json has "http://localhost:3000" set:
* blogs__element-coverage
* blogs__testing-redux-store
* blogs__vue-vuex-rest

**Comment by bahmutov** [comment]
Ok, the global start is no longer needed, since we test each project in isolation on Travis and on Circle. In each example folder we call `npm start` and then run Cypress. I will remove global start and related scripts now

**Comment by ninadkambli** [comment]
npm start command is not working in folder blogs__application-actions. Am getting below error

> blogs__page-actions@1.0.0 start D:\CypressExamples\cypress-example-recipes\examples\blogs__application-actions
> ../../node_modules/.bin/http-server -p 8888 --silent -c-1

'..' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! blogs__page-actions@1.0.0 start: `../../node_modules/.bin/http-server -p 8888 --silent -c-1`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the blogs__page-actions@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Ninad\AppData\Roaming\npm-cache\_logs\2020-01-20T17_02_17_328Z-debug.log

**Comment by jennifer-shehane** [comment]
See https://github.com/cypress-io/cypress-example-recipes/issues/404 for issue in comment above
